src/index.ts

```diff
@@ -131,8 +117,7 @@ export const plugin: Plugin = {
 /** The languages that are picked up by prettier. */
 export const languages: SupportLanguage[] | undefined = plugin.languages;
 /** The parsers object that is picked up by prettier. */
-export const parsers: { [parserName: string]: Parser } | undefined =
-  plugin.parsers;
+export const parsers = plugin.parsers;
```

Comment:
@fisker I would prefer using explicit types, so it runs into a compile-time error when something changes

```suggestion
export const parsers:
  | { [parserName: string]: Parser | (() => Promise<Parser>) }
  | undefined = plugin.parsers;
```

Reply:
I removed it because I don't know how to define types.

Reply:
Don't know, because we don't use TS.

Reply:
It's already defined as sync function, and we decide to keep the async support undocumented. https://github.com/prettier/prettier/pull/13211#issuecomment-1205047641